### ISSUE TITLE
No warning shown for duplicate attributes such as 'class'

### ISSUE THREAD

**Issue by drbcode** [dart-lang/angular_analyzer_plugin#648]
This component template doesn't show any warning while editing:

```html
<material-button class="use-session-btn" class="blue">Use</material-button>
```

During compile time we see this error:
`ParseErrorLevel.FATAL: Found multiple attributes with the same name: class.`

**Comment by MichaelRFairhurst** [comment]
Thanks for filing!

You're right. Our old parser would reject this before we even started to analyze, so we never implemented this. On the new parser, it'd be a really good & easy thing to catch!
